utils/test-stack-remover.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# The CI for this repo can sometimes leave some stacks lying around - every couple of months we need to clean them up.
+# This script will delete any stacks that are in DELETE_FAILED state, that contain the word _test_ in their name, and
+# that are not autoscaling stacks (these are child stacks that will be deleted when their parent is deleted)
+
+# no -u, as we rely on the $ROLE_ARN variable being passed in
```

Comment:
Shouldn't this be the ideal use case for `set -u`? If the `ROLE_ARN` not in the env, it will error, if it is not, it will not. Then you're prevented from unintentionally invoking the aws cloudformation command with an empty argument.

I ran shellcheck locally on this with `set-u` and it didn't find any problems.

Reply:
oh duh, onbiously 🤦